packages/react-packages/link/src/Link.tsx

```diff
@@ -33,6 +33,12 @@ export const Link = forwardRef<HTMLAnchorElement, LinkProps>(
     },
     ref
   ) => {
+    const iconSize: Record<LinkSize, IconProps['size']> = {
+      small: 'xs',
+      medium: 's',
+      large: 'l',
```

Comment:
should be this: no?
```suggestion
    const iconSize: Record<LinkSize, IconProps['size']> = {
      small: 's',
      medium: 'm',
      large: 'l',
```
also shuold it be move it to /constents?